people/Dethelly/page.en.md

```diff
@@ -56,13 +56,13 @@ That night, Sugar actually deleted her contact.
 The second day, a Twitter user who claimed to be Sugar's high school classmate brought her some random thoughts on behalf of Sugar.
 Then A requested them to look after her.
 
-Later, Sugar began her winter vacation, and got her diagnosis certificate of gender dysphoria in Shenzhen Kangning Hospital —— This was only known by A after her death.
+Later, Sugar began her winter holiday, and got her diagnosis certificate of gender dysphoria in Shenzhen Kangning Hospital —— This was only known by A after her death.
```

Comment:
May I ask why?

Reply:
I had British English in mind. My apologies, it should stay at vacation.